[CI] DriverTests testProfileAndStatusFinishInOneRound failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #7669 / almalinux-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/xu24vy4efso4i)
- [elasticsearch-periodic #7598 / graalvm-ce17_checkpart3_java-matrix](https://gradle-enterprise.elastic.co/s/5hohwiagenbbe)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:compute:test" --tests "org.elasticsearch.compute.operator.DriverTests.testProfileAndStatusFinishInOneRound" -Dtests.seed=E8DB241CEC4BBFB6 -Dtests.locale=id -Dtests.timezone=Atlantic/St_Helena -Druntime.java=24
```

**Applicable branches:**
8.18

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.compute.operator.DriverTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testProfileAndStatusFinishInOneRound),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: <80192L>
     but: was <681632L>
```

**Issue Reasons:**
- [8.18] 2 failures in test testProfileAndStatusFinishInOneRound (0.8% fail rate in 254 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/DriverTests.java
+++ b/x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/DriverTests.java
@@ -49,6 +49,7 @@ import java.util.function.LongSupplier;
 
 import static org.hamcrest.Matchers.either;
 import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
 
 public class DriverTests extends ESTestCase {
     /**
@@ -90,13 +91,13 @@ public class DriverTests extends ESTestCase {
         assertThat(driver.status().status(), equalTo(DriverStatus.Status.DONE));
         assertThat(driver.status().started(), equalTo(startEpoch));
         long sumRunningTime = tickTime * (nowSupplier.callCount - 1);
-        assertThat(driver.status().cpuNanos(), equalTo(sumRunningTime));
-        assertThat(driver.status().iterations(), equalTo((long) inPages.size()));
+        assertThat(driver.status().cpuNanos(), greaterThanOrEqualTo(sumRunningTime)); // GTE to work around not having #123290
+        assertThat(driver.status().iterations(), greaterThanOrEqualTo((long) inPages.size()));// GTE to work around not having #123290
 
         logger.info("profile {}", driver.profile());
-        assertThat(driver.profile().tookNanos(), equalTo(waitTime + sumRunningTime));
-        assertThat(driver.profile().cpuNanos(), equalTo(sumRunningTime));
-        assertThat(driver.profile().iterations(), equalTo((long) inPages.size()));
+        assertThat(driver.profile().tookNanos(), greaterThanOrEqualTo(waitTime + sumRunningTime));
+        assertThat(driver.profile().cpuNanos(), greaterThanOrEqualTo(sumRunningTime));
+        assertThat(driver.profile().iterations(), greaterThanOrEqualTo((long) inPages.size()));
     }
 
     /**

